sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -1050,6 +1050,9 @@ def check_sparse_input(EstimatorClass, X, X_sparse, y):
     assert_array_almost_equal(sparse.feature_importances_,
                               auto.feature_importances_)
 
+    assert_array_equal(sparse.predict(X_sparse), dense.predict(X))
+    assert_array_equal(dense.predict(X_sparse), sparse.predict(X))
```

Comment:
For floating point values this can vary: for instance if there are upcasts from `float` to `double` you don't always get the exact same values depending on the compilers and architecture.